hhs_hosp/delphi_hhs/run.py

```diff
@@ -79,39 +80,56 @@ def run_module():
         dfs.append(pd.DataFrame(response['epidata']))
     all_columns = pd.concat(dfs)
 
+    geo_mapper = GeoMapper()
+
     for sig in SIGNALS:
-        create_export_csv(
-            make_signal(all_columns, sig),
-            params["export_dir"],
-            "state",
-            sig
-        )
+        state = geo_mapper.add_geocode(make_signal(all_columns, sig),
+                                       "state_id", "state_code",
+                                       from_col="state")
+        for geo in GEOS:
```

Comment:
should tests be added for this logic? In this case I don't feel too strongly since they're just calling replace/add geocode and csv export which are both already tested, so going to approve regardless.

Reply:
...yes, especially considering how much time it took me to debug that section